src/EditorFeatures/Core.Wpf/NavigateTo/NavigateToItemDisplay.cs

```diff
@@ -128,7 +129,7 @@ public void PreviewItem()
                 return;
             }
 
-            var workspace = document.Project.Solution.Workspace;
+            var workspace = document.Workspace;
```

Comment:
📝 In theory I could have created a `NavigableSolution` record (and did originally), but that didn't seem like a valuable addition for this so I reduced it.

Reply:
Seems reasonable.